src/alg_utils.jl

```diff
@@ -441,3 +447,9 @@ uses_uprev(alg::CKLLSRK54_3C, adaptive::Bool) = adaptive
 uses_uprev(alg::CKLLSRK95_4S, adaptive::Bool) = adaptive
 uses_uprev(alg::CKLLSRK95_4C, adaptive::Bool) = adaptive
 uses_uprev(alg::CKLLSRK95_4M, adaptive::Bool) = adaptive
+uses_uprev(alg::CKLLSRK54_3C_3R, adaptive::Bool) = adaptive
```

Comment:
it doesn't look like you ever use uprev?

Reply:
No,I a step fails then `uprev` will be used to revert back to original position.

Reply:
oh okay yes